### DISCUSSION TITLE
Styling

### DISCUSSION THREAD

**Discussion by MrJelly17** [matt8707/ha-fusion#399] [Ideas]
I wanted to suggest an idea for future updates. It would be great to see a feature for general styling options where users can customize things like paddings, margins, and other layout elements. This would add a lot of flexibility and customization to the project. Looking forward to seeing what you come up with in future updates! Keep up the great work! 🚀

**Comment by matt8707** [comment]
FR: Theming https://github.com/matt8707/ha-fusion/issues/106
